pgproto3/row_description.go

```diff
@@ -70,7 +70,7 @@ func (dst *RowDescription) Decode(src []byte) error {
 		if idx < 0 {
 			return &invalidMessageFormatErr{messageType: "RowDescription"}
 		}
-		fd.Name = src[rp : rp+idx]
+		fd.Name = string(src[rp : rp+idx])
```

Comment:
If you would prefer to keep `Name` as a `[]byte` instead of a `string`, this is all that's needed to fix the bug:
```golang
        name := src[rp : rp+idx]
        fd.Name = make([]byte, len(name))
        copy(fd.Name, name)
```